test/e2e_node/topology_manager_test.go

```diff
@@ -52,6 +53,7 @@ const (
 
 	minNumaNodes = 2
 	minCoreCount = 4
+	minSriovResource = 7
```

Comment:
my only doubt is about this value. We should probably document if it is a number we made up to mean "just enough" resources or if we computed them somehow - this applies also to my `minCoreCount`, hopefully `minNumaNodes` is self-explanatory :)

Reply:
I agree. We need to put in as pre-requisite of some sorts. Maybe add it to the documentation that is there ?

Reply:
Yes, agree this should be documented. This is a number that just sorta felt right, no real computation. For most of the systems I have been on, they all support 64 VFs or more, so 7 is way under that.